src/MCRun.jl

```diff
@@ -228,9 +248,9 @@ for atom, volume and rotation moves.
 Performs the selected move.   
 """
 function mc_step!(mc_state::MCState, pot, ensemble, a, v, r)
-    ran = rand(1:(a+v+r)) #choose move randomly
-    if ran <= a
-        mc_state = atom_move!(mc_state, ran, pot, ensemble)
+    ran_atom = rand(1:(a+v+r)) #choose move randomly
+    if ran_atom <= a
+        mc_state = atom_move!(mc_state, ran_atom, pot, ensemble)
```

Comment:
redefinition to avoid using the name ran

Reply:
that's fine